### PR TITLE
add login endpoint

### PR DESCRIPTION
**Pull request by JorgeMichelena**
-Changed default user to custom model located in users
-Added login endpoints from django-rest-auth

### CODE REVIEW

File: api/urls.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.urls import path, include
+
+urlpatterns = [
+    path('', include('rest_auth.urls')),
+]
```
Comment (jiss2891) [inline api/urls.py]: can you add a newline here?

File: target/settings.py [hunk 2]
```diff
@@ -37,6 +37,13 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    'rest_framework',
+    'rest_framework.authtoken',
+    'rest_auth',
+
```
Comment (jiss2891) [inline target/settings.py]: why did you left those blank lines?

File: api/admin.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (kaozdl) [inline api/admin.py]: the api does not need an admin model, this file is safely deletable

File: api/models.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```
Comment (kaozdl) [inline api/models.py]: the api does not need models, this file is safely deletable

File: api/urls.py (hunk 1 above)
Comment (kaozdl) [inline api/urls.py]: add a newline here

File: target/settings.py [hunk 5]
```diff
@@ -37,6 +37,13 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
```
Comment (kaozdl) [inline target/settings.py]: maybe adding a comment here for separating third partys and your stuff?

**Pull request merged**